src/status_im2/contexts/chat/messages/list/events.cljs

```diff
@@ -1,12 +1,12 @@
-(ns status-im.chat.models.message-list
-  (:require ["functional-red-black-tree" :as rb-tree]
-            [status-im.constants :as constants]
+(ns status-im2.contexts.chat.messages.list.events
+  (:require [react-native.red-black-tree :as rb-tree]
```

Comment:
I think we should stick to our convention of preferring explicit namespace aliases, so in this case: `red-black-tree` instead of `rb-tree`.